assets/app/scripts/app.js

```diff
@@ -248,7 +248,16 @@ angular
         templateUrl: 'views/util/logout.html',
         controller: 'LogoutController'
       })
-
+      // legacy redirects
+      .when('/createProject', {
+        redirectTo: '/create-project'
```

Comment:
We should throw deprecation warnings when we hit these, especially so we catch our own if they pop up.

`console.warn("Deprecated: used createProject route instead of create-project");`